config/webpack.config.dev.js

```diff
@@ -65,6 +65,9 @@ module.exports = {
     publicPath: '/'
   },
   resolve: {
+    // This allows you to set a root for where webpack should look for modules.
+    // This enables you to use absolute imports from the root.
+    root: path.resolve(env['process.env.NODE_PATH']),
```

Comment:
Will this work for multiple paths?
From Node docs:

>If the NODE_PATH environment variable is set to a colon-delimited list of absolute paths, then Node.js will search those paths for modules if they are not found elsewhere. (Note: On Windows, NODE_PATH is delimited by semicolons instead of colons.)